tox.ini

```diff
@@ -172,18 +177,22 @@ commands_pre =
 
   test-auto-instrumentation: pip install {toxinidir}/opentelemetry-auto-instrumentation
 
-  example-app: pip install {toxinidir}/opentelemetry-auto-instrumentation {toxinidir}/ext/opentelemetry-ext-requests {toxinidir}/ext/opentelemetry-ext-wsgi {toxinidir}/ext/opentelemetry-ext-flask {toxinidir}/docs/examples/opentelemetry-example-app
+  example-app: pip install {toxinidir}/opentelemetry-auto-instrumentation {toxinidir}/ext/opentelemetry-ext-utils {toxinidir}/ext/opentelemetry-ext-requests {toxinidir}/ext/opentelemetry-ext-wsgi {toxinidir}/ext/opentelemetry-ext-flask {toxinidir}/docs/examples/opentelemetry-example-app
 
-  getting-started: pip install -e {toxinidir}/opentelemetry-auto-instrumentation -e {toxinidir}/ext/opentelemetry-ext-requests -e {toxinidir}/ext/opentelemetry-ext-wsgi -e {toxinidir}/ext/opentelemetry-ext-flask
+  getting-started: pip install -e {toxinidir}/opentelemetry-auto-instrumentation -e {toxinidir}/ext/opentelemetry-ext-utils -e {toxinidir}/ext/opentelemetry-ext-requests -e {toxinidir}/ext/opentelemetry-ext-wsgi -e {toxinidir}/ext/opentelemetry-ext-flask
 
   grpc: pip install {toxinidir}/ext/opentelemetry-ext-grpc[test]
 
+  wsgi,flask,django,asgi: pip install {toxinidir}/ext/opentelemetry-ext-utils
```

Comment:
since this is used in pretty much every extension: maybe we can do a broader match for tox? IIRC tox is substring match, so you could may just use the string "ext":

`ext: ...opentelemetry-ext-utils`

Reply:
good idea, changed it. also renamed the package